agent/command/expansion_write.go

```diff
@@ -48,10 +49,10 @@ func (c *expansionsWriter) Execute(ctx context.Context,
 	expansions := map[string]string{}
 	for k, v := range conf.Expansions.Map() {
 		_, ok := conf.Redacted[k]
-		// Users should not be able to use the global github token expansion
-		// as it can result in the breaching of Evergreen's GitHub API limit.
-		// Likewise with AWS expansions.
-		if (ok && !c.Redacted) || utility.StringSliceContains(ExpansionsToRedact, k) {
+		// Users should not be able to use the global github token
+		// expansion as it can result in the breaching of Evergreen's
+		// GitHub API limit. Likewise with AWS expansions.
+		if (ok && !c.Redacted && !isPerfProject(conf)) || utility.StringSliceContains(ExpansionsToRedact, k) {
```

Comment:
This took me some time to parse, since the comment refers only to I think the second half of the if statement (redacting global expansions) but the first half does the actual verification of private variables, right? Can you update this comment to reflect the current state?

Reply:
Done!

Reply:
I just simplified it a bit.